src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -52,6 +61,51 @@ object EMD extends DebugEnhancedLogging {
     }
   }
 
+  def addAgreementFields(emd: EasyMetadata)(implicit s: Settings): Unit = {
+    val agreementPath = new File(s.bagitDir, depositorInfoDir.resolve("agreements.xml").toString)
+    if (agreementPath.exists) {
+      val agreementsXml = XML.loadFile(agreementPath)
+      if (BooleanUtils.toBoolean((agreementsXml \\ "depositAgreementAccepted").text)) {
```

Comment:
i would like a WARN in the else of this case, so if the depositAgreementAccepted is false